examples/pipelines/simple_pipeline/components/image_filtering/src/main.py

```diff
@@ -0,0 +1,39 @@
+"""
+This component filters images of the dataset based on image size (minimum height and width).
+"""
+import logging
+from typing import Dict
+
+from datasets import Dataset
+
+from express.dataset import FondantComponent
+from express.logger import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+class ImageFilterComponent(FondantComponent):
+    """
+    Component that filters images based on height and width.
+    """
+
+    @classmethod
+    def process(cls, dataset: Dataset, args: Dict) -> Dataset:
```

Comment:
Can we swtich over to using the t.Dict everywhere to be consistent. Also, if you're the `Dict` from the `typing` module, it's better to define the datatypes inside of it